test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -734,7 +734,6 @@ def _test_large_scale_exporter(
                 tuple(ctx.paths),
                 onnx_model,
             )
-
```

Comment:
I recall the two tests related to `_test_large_scale_exporter` are using workaround FakeTensor export. I think they should be refactored as well.

cc @wschin

Reply:
Good point.
They cannot run with this PR yet because of the limitation [1] described above. In the next PR it will be resolved and that test refactored

Reply:
Good point.
They cannot run with this PR yet because of the limitation [1] described above. In the next PR it will be resolved and that test refactored